src/screens/LandingScreen/SpeakerSection.js

```diff
@@ -294,6 +336,48 @@ const SpeakerSection = () => {
                     icon="github"
                     title="Juan on GitHub"
                   />
+
+                  <Box
+                    sx={{
+                      fontSize: '12px',
+                      ml: '35%',
+                      mb: '0',
+                      backgroundColor: '#EC3750',
+                      backgroundImage: 'at left top, rgb(255,140,55)',
+                      padding: '3px 5px',
+                      borderRadius: '5px',
+                      color: '#fff'
+                    }}
+                  >
+                    <a
+                      style={{
+                        textDecoration: 'none',
+                        display: 'flex',
+                        color: '#fff'
+                      }}
+                      href="https://google.com/"
```

Comment:
Juan's Link goes here

Reply:
https://calendar.google.com/calendar/u/0/r/eventedit?uid=5f4212d7db36e40015f240cf&location=http://hacknitr.tech/live&ctz=Asia/Calcutta&text=Juan%20Pablo%20Flores%20X%20HackNITR%202.0&dates=20210321T220000/20210321T230000&details=This%20is%20the%20reminder%20for%20Juan%20Pablo%20Flores%27s%20session.%20HackNITR%202.0%20will%20be%20conducted%20entirely%20on%20Airmeet%20and%20the%20join%20link%20will%20be%20active%2006%20hrs%20before%20the%20event.%0A%0AJoin%20Link%3A%20http%3A//hacknitr.tech/live